src/components/HistoryTable/hooks/useTestResults.test.tsx

```diff
@@ -1,86 +1,32 @@
+import { useHistoryTableTestHook, ProviderWrapper } from "./test-utils";
 import useTestResultsActual from "./useTestResults";
 
-interface ProviderProps {
-  children: React.ReactNode;
-  mocks?: MockedResponse[];
-}
-
-const ProviderWrapper: React.FC<ProviderProps> = ({ children, mocks = [] }) => (
-  <MockedProvider mocks={mocks}>
-    <HistoryTableProvider>{children}</HistoryTableProvider>
-  </MockedProvider>
-);
-
-/** useMergedHookRender takes the useTestResults and useHistoryTable hooks
- * and combines them into a shared hook which can be rendered under the same wrapper context
- * and can be used together */
-const useMergedHookRender = ({ row }) => {
-  const useTestResults = useTestResultsActual(row);
-  const useHistoryTable = useHistoryTableActual();
-  return {
-    useTestResults,
-    useHistoryTable,
-  };
-};
-
-// This is a sanity check to ensure the useMergedHookRender hook is working as expected
-describe("useMergedHookRender - sanity check", () => {
-  it("should return the correct hooks", () => {
-    const { result } = renderHook(() => useMergedHookRender({ row: 0 }), {
-      wrapper: ProviderWrapper,
-    });
-    expect(result.current.useTestResults).toStrictEqual({
-      getTaskMetadata: expect.any(Function),
-    });
-    expect(result.current.useHistoryTable).toStrictEqual({
-      processedCommitCount: 0,
-      fetchNewCommit: expect.any(Function),
-      getItem: expect.any(Function),
-      isItemLoaded: expect.any(Function),
-      getItemHeight: expect.any(Function),
-      toggleRowSizeAtIndex: expect.any(Function),
-      hasNextPage: false,
-      hasPreviousPage: false,
-      historyTableFilters: [],
-      setHistoryTableFilters: expect.any(Function),
-      processedCommits: [],
-      visibleColumns: [],
-      addColumns: expect.any(Function),
-      nextPage: expect.any(Function),
-      previousPage: expect.any(Function),
-      currentPage: 0,
-      pageCount: 0,
-      columnLimit: 7,
-      commitCount: 10,
-    });
-  });
-});
-
 describe("useTestResults", () => {
   it("should return an empty map when nothing is loaded", () => {
-    const { result } = renderHook(() => useMergedHookRender({ row: 0 }), {
-      wrapper: ({ children }) => ProviderWrapper({ children }),
-    });
-    expect(result.current.useTestResults).toStrictEqual({
+    const { result } = renderHook(
+      () => useHistoryTableTestHook(useTestResultsActual, [0]),
+      {
+        wrapper: ({ children }) => ProviderWrapper({ children }),
+      }
+    );
+    expect(result.current.hookResponse).toStrictEqual({
       getTaskMetadata: expect.any(Function),
     });
   });
   it("should return the default state when there is no valid data for a row", () => {
-    const { result } = renderHook(() => useMergedHookRender({ row: 0 }), {
-      wrapper: ({ children }) => ProviderWrapper({ children }),
-    });
+    const { result } = renderHook(
+      () => useHistoryTableTestHook(useTestResultsActual, [0]),
```

Comment:
Why don't we render `useTestResultsActual` directly instead of passing it into useHistoryTableTestHook?